docs/navigation-configuration.md

```diff
@@ -6,23 +6,24 @@ Navigation parameters allow you to specify routing configuration, set the appear
 
 The image shows the elements of Luigi navigation: 
 
-1. Top navigation, where the main navigation path is displayed.
-2. Side navigation, where the applications are defined.
-3. Main content window, where the micro front-end is rendered.
+1. Top navigation, which displays the main navigation path.
+2. Side navigation, which displays the defined applications.
+3. Main content window, which renders the micro frontend.
  
 
-![Navigation layout](assets/navigation-structure.png)
+![Navigation elements](assets/navigation-structure.png)
```

Comment:
Not sure if `elements` is a good word here. I kinda liked `layout` better :)

Reply:
I changed it back to layout :)